adoc/retail_chap_next.adoc

```diff
@@ -61,10 +61,10 @@ If you need further information or support, try one of these options.
 
 For {susemgr} documentation, visit https://www.suse.com/documentation/suse-manager-3/
 
-For legacy {slepos} docuemntation, visit https://www.suse.com/documentation/slepos11/index.html
-Note, however, that {smr} documentation supercedes this information.
+For legacy {slepos} documentation, visit https://www.suse.com/documentation/slepos11/index.html
```

Comment:
Even if the sentence ends with a filename or Web resource entity, I'd finish it with a full stop.  I frankly admit that I do not know what all our style guides wants us to do in those cases...